generated/nifake/tests/test_session.py

```diff
@@ -477,35 +477,35 @@ def test_get_attribute_error(self):
                 assert e.code == test_error_code
                 assert e.description == test_error_desc
 
-    # Error descriptions
-
-    def test_get_error_description_get_error_message(self):
+    '''
```

Comment:
Why is this commented out?
Add a TODO that explains why and when it needs to be put back in.

Reply:
Fixed